1/backend/src/resolvers.js

```diff
@@ -1,67 +1,268 @@
 const jwt = require('jsonwebtoken');
 const fs = require('fs');
+const {uuid} = require('uuidv4');
 
-data = require('./data');
+let data;
 
 let SECRET_KEY = fs.readFileSync('./src/key/secret.key', 'utf8');
 
 const resolvers = {
     Query: {
-        todos: () => JSON.parse(JSON.stringify(data.todos))
+        todos: async (parent, args, context) => {
+            // JSON.parse(JSON.stringify(data.todos))
+            if (args.limit != "" && args.limit != null) {
```

Comment:
First, I think these checks can be removed/refactored. Second, what happens in the `else` branch?